vscode/src/test/suite/ruby/chruby.test.ts

```diff
@@ -318,4 +318,36 @@ suite("Chruby", () => {
       return chruby.activate();
     });
   });
+
+  test("Uses closest Ruby if the version specified in .ruby-version is not installed (patch difference)", async () => {
+    fs.writeFileSync(path.join(workspacePath, ".ruby-version"), "ruby '3.3.3'");
+
+    const chruby = new Chruby(workspaceFolder, outputChannel, async () => {});
```

Comment:
We repeat some very similar setup in many tests in this file, perhaps we can extract out some common behaviour?

Reply:
I extracted the assertions into a function.